ImperatorToCK3/Source/CK3/Titles/Title.h

```diff
@@ -27,31 +27,24 @@ class Title
 		mappers::ProvinceMapper& provinceMapper,
 		mappers::CoaMapper& coaMapper);
 
-	[[nodiscard]] const auto& getTitleName() const { return titleName; }
-	[[nodiscard]] const auto& getHistoryCountryFile() const { return historyCountryFile; }
-	[[nodiscard]] const auto& getLocalizations() const { return localizations; }
-	[[nodiscard]] const auto& getCoa() const { return coa; }
-	[[nodiscard]] auto getCapitalCounty() const { return capitalCounty; }
-	[[nodiscard]] const auto& getImperatorCountry() const { return imperatorCountry; }
+	int holder = 0;
```

Comment:
I know we talked about this, and I can't point to any particular reason why it's bad, but it does rub me the wrong way. :-D

Reply:
because holder 0 is actually a legit ID for a person in CK3 saves (negatives would not be legit maybe)?
If you're talking about initialization, ints should be inited. If not, VS compains when instancing the whole class without a constructor that sets it.

Reply:
I actually used 0 because it means "no character". `holder = 0` used in a title history removes the holder:
```
#Seljuk Turks
	1063.9.4 = {
		holder = 3040
	}
	1072.11.1 = {
		holder = 3042
	}
	1092.11.1 = {
		holder = 144012
	}
	1094.1.1 = {
		holder = 144013
	}
	1105.3.1 = {
		holder = 144014
	}
	1118.4.18 = {
		holder = 144015
	}
	1157.5.8 = {
		holder = 0 # End of the Seljuk Turks
	}
```
I need to change it to "0" string, though. CK3 chararacter IDs are strings.